io.github.pemsley.coot.yaml

```diff
@@ -0,0 +1,407 @@
+        url: https://github.com/cctbx/ccp4io/archive/b58c4fb68902e4e6a58f4a585d0722e542516076.tar.gz
+        sha256: f1edc5a830cd4a078eae700e14b1d89612fb7a12318094363642340aafe41af6
+
+  - name: clipper4coot
+    buildsystem: autotools
+    config-opts:
+      - --enable-mmdb
+      - --enable-ccp4
+      - --enable-cif
+      - --enable-minimol
+      - --enable-cns
+      - --enable-shared
+      - --disable-static
+    build-options:
+      cxxflags: "-g -O2 -fno-strict-aliasing -Wno-narrowing"
+      env:
+        LDFLAGS: "-L/app/lib"
+        CPPFLAGS: "-I/app/include"
+    sources:
+      - type: archive
+        url: https://www2.mrc-lmb.cam.ac.uk/personal/pemsley/coot/dependencies/clipper-2.1.20180802.tar.gz
+        sha256: 7c7774f224b59458e0faa104d209da906c129523fa737e81eb3b99ec772b81e0
+      - type: patch
+        path: clipper-configure-2.patch
+        sha256: 3cf0a68163451773e9764c11c740fcbd1a91daf9d5782d94049b90f3cd1fe5ae
+
+  - name: libgd
+    buildsystem: autotools
+    config-opts:
+      - --without-x
+      - --without-xpm
+    sources:
+      - type: archive
+        url: https://github.com/libgd/libgd/releases/download/gd-2.3.3/libgd-2.3.3.tar.xz
+        sha256: 3fe822ece20796060af63b7c60acb151e5844204d289da0ce08f8fdf131e5a61
+      - type: patch
+        path: gd.h.patch
+        sha256: 1015f6e125f139a1e922ac4bc2a18abbc498b0142193fa692846bf0f344a3691
+
+  - name: raster3d
+    buildsystem: simple
+    config-opts:
+      - prefix=/app
+    build-options:
+      cxxflags: "-g -O2 -fno-strict-aliasing -Wno-narrowing"
+      env:
+        LDFLAGS: "-L/app/lib"
+        CFLAGS: "-I/app/include"
+    build-commands:
+      - sed -i 's|prefix  = /usr/local|prefix  = /app|' Makefile.template
+      - sed -i 's|INCDIRS  =	-I/usr/include -I/usr/local/include|INCDIRS  = -I/app/include|' Makefile.template
+      - sed -i 's|LIBDIRS  =	-L/usr/local/lib|LIBDIRS  = -L/app/lib|' Makefile.template
+      - sed -i 's|mandir  = $(prefix)/man/manl|mandir  = /app/man/manl|' Makefile.template
+      - make linux
+      - make all
+      - make install
+    sources:
+      - type: archive
+        url: http://www.bmsc.washington.edu/raster3d/Raster3D_3.0-7.tar.gz
+        sha256: f566b499fee341db3a95229672c6afdbdb69da7faabdbe34f6e0d332d766160c
+
+  - name: ssm
+    buildsystem: autotools
+    config-opts:
+      - --enable-ccp4
+      - --enable-shared
+      - --disable-static
+    build-options:
+      cflags: -fPIC
+    sources:
+      - type: archive
+        url: https://www2.mrc-lmb.cam.ac.uk/personal/pemsley/coot/dependencies/ssm-1.4.tar.gz
+        sha256: 56e7e64ed86d7d9ec59500fd34f26f881bdb9d541916d9a817c3bfb8cf0e9508
+
+  - name: boost
+    buildsystem: simple
+    build-commands:
+      - ./bootstrap.sh --with-icu=/app --prefix=/app --libdir=/app/lib
+      - ./b2 install --prefix=/app --libdir=/app/lib -d2 -j$(nproc) threading=multi link=shared,static
+    config-opts:
+      - --without-libraries=[python, mpi]
```

Comment:
@pemsley 
We needn't to install `mpi`(Boost MPI Library), right?

Reply:
I checked your build script on coot website.

There is no `mpi` in the script, so I deleted it.